robot/robot/src/by/teachmeskills/robot/Run.java

```diff
@@ -0,0 +1,49 @@
+package by.teachmeskills.robot;
+
+import by.teachmeskills.robot.hands.SonyHand;
+import by.teachmeskills.robot.hands.ToshibaHand;
+import by.teachmeskills.robot.heads.SamsungHead;
+import by.teachmeskills.robot.heads.SonyHead;
+import by.teachmeskills.robot.heads.ToshibaHead;
+import by.teachmeskills.robot.legs.SamsungLeg;
+import by.teachmeskills.robot.legs.SonyLeg;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Run {
+    public static void main(String[] args) {
+        Robot android = new Robot(new SonyHead(10), new SonyHand(10), new SonyLeg(10), "Android");
+        android.action();
+        Robot bender = new Robot(new SamsungHead(20), new ToshibaHand(2), new SamsungLeg(10), "Bender");
+        bender.action();
+        Robot bostonDynamic = new Robot(new ToshibaHead(4), new SonyHand(3), new SamsungLeg(30), "Boston Dynamic");
+        bostonDynamic.action();
+        {
+            List<Robot> robots = new LinkedList<>();
+            robots.add(android);
+            robots.add(bender);
+            robots.add(bostonDynamic);
+            int max = robots.get(0).getPrice();
+            String nameMax = robots.get(0).getName();
+            for (Robot robot : robots) {
+                if (robot.getPrice() > max) {
+                    max = robot.getPrice();
+                    nameMax = robot.getName();
+                }
+            }
+            System.out.println("Max price\t" + nameMax);
+        }
```

Comment:
create a separate class with a method that will do this, we are now using OOP =)